src/reducer.ts

```diff
@@ -66,10 +67,11 @@ export interface AppState {
     arbeidsliste: any;
     enhettiltak: EnhettiltakState;
     features: FeaturesState;
-    veiledergrupperLagretFilter: VeiledergrupperLagretFilterState;
-    mineFilter: MineFilterState;
-    mineFilterMinOversikt: MineFilterUIState;
-    mineFilterEnhetensOversikt: MineFilterUIState;
+    veiledergrupperLagretFilter: LagretFilterState;
+    mineFilter: LagretFilterState;
+    lagretFilterMinOversikt: FilterUIState;
```

Comment:
these should probably still be `mineFilterMinOversikt`, `mineFilterEnhetensOversikt` and `mineFilterVeilederOversikt`, since they are not including veiledergrupper?

Reply:
And also: FilterUIState should be LagretFilterUIState

Reply:
Yep, I agree...will refactor

Reply:
Pls check again...for me it seems that maybe we can improve naming in reducer bit more..

Reply:
What do you mean?

Reply:
maybe those two can be more consistent:
```
veiledergrupperLagretFilter: veiledergrupperLagretFilterReducer, 
mineFilter: mineFilterReducer,
```